app/components/listing/feedback/FeedbackModal.jsx

```diff
@@ -103,9 +97,6 @@ const FeedbackModal = ({ service, resource, closeModal }) => {
         <img src={images.icon('feedback-blue-header')} alt="feedback" />
         <span>Share your Feedback</span>
       </div>
-      <div className={styles.feedbackSubheader}>
-        The team usually replies within a day.
-      </div>
```

Comment:
This doesn't seem like it has anything to do with the change to make the feedback box optional. Was this accidentally deleted or was this intentional?

If this was intentional, then please at least make a mention of this change in your PR description. Currently, the PR title only says that this fixes a "bug" in the submit button, and the PR description makes no mention of other changes being made. If you don't explicitly mention these kinds of things, then it becomes difficult for a code reviewer to understand whether a change was intentional or accidental.

Reply:
Thank you for bringing this up, just updated description.